CHANGELOG.md

```diff
@@ -7,14 +7,14 @@
 
 ### Fixes
 
-
 ### Chores
 
 - Updated `cardano-wallet` to v2022-10-06 ([PR 3045](https://github.com/input-output-hk/daedalus/pull/3045))
 - Changed Japanese translations in delegate screen ([PR 3044](https://github.com/input-output-hk/daedalus/pull/3044))
 - Updated trezor-connect to v9 ([PR 3038](https://github.com/input-output-hk/daedalus/pull/3038))
 - Added analytics data collection ([PR 2927](https://github.com/input-output-hk/daedalus/pull/2927), [PR 2989](https://github.com/input-output-hk/daedalus/pull/2989), [PR 3003](https://github.com/input-output-hk/daedalus/pull/3003), [PR 3028](https://github.com/input-output-hk/daedalus/pull/3028))
 - Updated LedgerJS to 5.1.0 ([PR 3036](https://github.com/input-output-hk/daedalus/pull/3036))
+- Fixed locale in Storybook ([PR 3042](https://github.com/input-output-hk/daedalus/pull/3042))
```

Comment:
We have a rule (unfortunately probably unwritten) that we add new entries at the top, so this one should be placed above the ``Updated `cardano-wallet` to v2022-10-06`` entry 😉

Reply:
Actually... Should this even be added to the changelog? I think the changelog is about Daedalus only...

Reply:
I think it's fine to list it as chore. So far we've been listing all tickets in the changelog with no exceptions. It is very unlikely that any change affects the production code, but if it does, it will be easier to find a PR if it's listed in changelog.